modules/local/mtx_to_seurat.nf

```diff
@@ -19,10 +19,10 @@ process MTX_TO_SEURAT {
 
     script:
     def aligner = params.aligner
-    if (params.aligner == "cellranger") {
-        matrix   = "matrix.mtx.gz"
-        barcodes = "barcodes.tsv.gz"
-        features = "features.tsv.gz"
+    if (params.aligner == "cellranger" || params.aligner == "cellrangerarc") {
```

Comment:
We can follow up on that separately, but I think for ATAC seq we should generate a [MuData](https://mudata.readthedocs.io/en/latest/) (h5mu) object with both ATAC and GEX data.

Reply:
That is true. Also in the MTX_TO_H5AD.

Reply:
actually, MuData would only be in the mtx_to_h5ad... Not sure what the best representation of multimodal single-cell data is in the Seurat/Bioconductor world is. 

But I'd suggest to leave it for now, open an issue and tackle this in a follow-up PR.